sorc/machine-setup.sh

```diff
@@ -121,8 +135,10 @@ elif [[ -d /lfs3 ]] ; then
     # We are on NOAA Jet
     if ( ! eval module help > /dev/null 2>&1 ) ; then
         echo load the module command 1>&2
+        # shellcheck disable=SC1090
         source /apps/lmod/lmod/init/$__ms_shell
     fi
+    # shellcheck disable=
```

Comment:
Move this up a line inside the if-block like the command it is undoing.

Reply:
This is a feature of shellcheck directives.  The next line of code cannot be a block ending or partial statement.  See [this note](https://www.shellcheck.net/wiki/SC1123).  I could move the directive before the if block to match tiers, but that may be confusing as to what it is there for.